R/fix_sc_dates.R

```diff
@@ -41,3 +42,28 @@ fix_sc_end_dates <- function(start_date, end_date, period) {
 
   return(end_date)
 }
+
+
+
+
+#' Fix sc end dates
+#'
+#' @description Fix social care end dates when the end date is earlier than the
+#' start date. Set this to the end of the fyear
+#'
+#' @param start_date A vector containing dates.
+#' @param end_date A vector containing dates.
+#' @param period Social care latest submission period.
+#'
+#' @return A date vector with replaced end dates
+fix_sc_missing_end_dates <- function(end_date, period_end) {
+  # Fix sds_end_date is earlier than sds_start_date by setting end_date to be
+  # the end of financial year
```

Comment:
Does this comment apply here? or are we just changing this to end of period id the end date is earlier than the start date?

Reply:
This is just a simpler way of doing the same thing as it was before. 
Before it took period and worked out period start and end dates from that, whereas I'm just getting them from the data extraction